sandbox/server.js

```diff
@@ -65,8 +65,15 @@ server = http.createServer(function (req, res) {
     });
   } else {
     res.writeHead(404);
-    res.end('<h1>404 Not Found</h1>');
+    res.end("<h1>404 Not Found</h1>");
```

Comment:
Linting rules should enforce single-quotes- does linting run on commit ?

Reply:
Doesn't seem like linting covers those files, the whole file is filled with eslint errors for me, fixed.